cmake/recipes/external/cgal.cmake

```diff
@@ -34,8 +34,10 @@ function(cgal_import_target)
     include(boost)
 
     ignore_package(GMP 5.0.1)
-    set(GMP_INCLUDE_DIR "")
+    set(GMP_INCLUDE_DIR ${gmp_INCLUDE_DIR})
     set(GMP_LIBRARIES gmp::gmp)
+    set(GMPXX_INCLUDE_DIR ${GMP_INCLUDE_DIR})
```

Comment:
I wonder if we should add `ignore_package(GMPXX 5.0.1)` as well just to be safe?

Reply:
Do you know what this is doing? If it makes sense to add, then happy to.

Reply:
(adding this doesn't seem to hurt)

Reply:
`ignore_package()` is a function I wrote in this `cgal.cmake`. Basically it will add an empty `GMPXXConfig.cmake` file, thus preventing CGAL from using its own `FindGMPXX.cmake` file (and subsequently adding `/usr/local/include` to your include paths).